front/.gitignore

```diff
@@ -39,4 +39,7 @@ next-env.d.ts
 
 #sst
 .sst
-.sst/
\ No newline at end of file
+.sst/
```

Comment:
Is this intentional? Or can we do
```
**/*.sst
```

Reply:
It's intentional, because there is a specific `.sst` folder 😅 If we did **, we might accidently ignore all .sst files anywhere 👀